src/alpine/3.8/helix/amd64/DockerFile

```diff
@@ -0,0 +1,56 @@
+FROM python:2.7.15-alpine3.8
```

Comment:
The source of CI failing is because the standard convention is to name this `Dockerfile` with a lower cased 'f' not upper cased.  If you don't want to use the standard convention, you would need to specify the full path in the manifest.  I would recommend using the standard convention.

Reply:
Thanks, obvious in retrospect... :)